ai-brain/fix-failed-prs.js

```
@@ -0,0 +1,525 @@
+    const prDetails = await getPRDetails(prNumber);
+    
+    if (!prDetails) {
+      console.log(`❌ Could not fetch PR details`);
+      return false;
+    }
+    
+    // Check if all checks are passing
+    const hasFailures = hasFailedChecks(prDetails);
+    const hasConflicts = hasConflicts(prDetails);
+    
+    if (!hasFailures && !hasConflicts && prDetails.mergeable === 'MERGEABLE') {
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)